infrastructure/async/src/main/java/tech/pegasys/teku/infrastructure/async/stream/AsyncStream.java

```diff
@@ -98,10 +92,6 @@ default AsyncStream<T> limit(final long count) {
     return slice(AsyncStreamSlicer.limit(count));
   }
 
```

Comment:
We may use `exceptional` and `takeWhile` in the future. It's a library methods. Not sure on this.